buildscripts/builddrivers.py

```diff
@@ -0,0 +1,287 @@
+                
+            os.chdir(work_dir)  # change back to the working directory
+
+    def build_extensions(self, dest, logfile):
+        """This takes care of getting the drivers' source files, building the drivers. 
+        If running locally, *dest* should be the root drive. Otherwise, *dest* should be None. 
+        In this case, remote_path must be defined such that the binaries will be copied 
+        to the designated destinations.
+        
+        :param  dest: either None (for remote builds) or the C:\ drive (for local builds)
+        :param  logfile: the name of the logfile
+        :outcome: the drivers and symbols will renamed and placed in the appropriate location(s)
+
+        """
+        work_dir = os.path.dirname(os.path.realpath(__file__))
+            
+        if self.download_source:
+            # This will download from the specified branch on GitHub repo and copy the source to the working directory
+            self.util.download_msphpsql_source(repo, branch)
+        else:
+            # This case only happens when building locally (interactive mode)
+            # because download_source must be True for remote builds
+            while True:
+                if self.source_path is None:
+                    source = input('Enter the full path to the Source folder: ')
+                else:
+                    source = input("Hit ENTER to reuse '" + self.source_path + "' or provide another path to the Source folder: ")
+                    if len(source) == 0:
+                        source = self.source_path
+
+                valid = True
+                if os.path.exists(source) and os.path.exists(os.path.join(source, 'shared')):
+                    # Checking the existence of 'shared' folder only, assuming
+                    # sqlsrv and/or pdo_sqlsrv are also present if it exists
+                    self.source_path = source
+                    break
+                    
+                print("The path provided is invalid. Please re-enter.")            
+                                
+            print('Copying source files from', source)
+                
+            os.system('ROBOCOPY ' + source + '\shared ' + work_dir + '\Source\shared /xx /xo ')
+            os.system('ROBOCOPY ' + source + '\sqlsrv ' + work_dir + '\Source\sqlsrv /xx /xo ')
+            os.system('ROBOCOPY ' + source + '\pdo_sqlsrv ' + work_dir + '\Source\pdo_sqlsrv /xx /xo ')
+                    
+        print('Start building PHP with the extension...')
+
+        self.util.build_drivers(self.make_clean, dest, logfile)
+
+        if dest is None:       
+            # This indicates the script is NOT running locally, and that 
+            # the drivers should be in the working directory
+
+            # Make sure drivers path is defined
+            if self.remote_path is None:
+                print('Errors: Drivers destination should be defined! Do nothing.')
```

Comment:
'Doing nothing.' sounds better, otherwise it reads like an instruction to the user, which is confusing.